packages/client/hmi-client/src/components/workflow/ops/calibrate-ensemble-ciemss/calibrate-ensemble-ciemss-operation.ts

```diff
@@ -11,7 +12,7 @@ export interface EnsembleCalibrateExtraCiemss {
 }
 
 export interface CalibrateEnsembleCiemssOperationState extends BaseState {
-	chartConfigs: string[][];
+	chartConfigs: Partial<ChartConfig>[];
```

Comment:
You may want to leave ensemble-related operators alone for the moment - I think Nelson is catching up with TA3 folks on what we want to do in this space, so not particularly worthwhile to get them into the latest when we don't precisely know what we want.

Reply:
Not longer touching these at the moment